include/pcapng_exporter/flexray.h

```diff
@@ -36,6 +36,7 @@ struct flexray_frame
 	uint8_t cc;
 
 	uint8_t data[254];
+	uint8_t symbol_length;
```

Comment:
In PCAP FlexRay symbol is defined as array of bytes, not just one value

https://www.tcpdump.org/linktypes/LINKTYPE_FLEXRAY.html